alt_requirements/requirements_tensorflow_sklearn.txt

```diff
@@ -3,4 +3,5 @@
 
 scikit-learn==0.19.1
 tensorflow==1.6.0
+scipy==1.0.0
```

Comment:
`scipy` latest release version is `1.1.0`. I create a PR#1153. https://pypi.org/project/scipy/#history